base/server/cms/src/com/netscape/cms/profile/common/EnrollProfile.java

```diff
@@ -549,8 +553,21 @@ public void submit(IAuthToken token, IRequest request)
             CMS.debug(e);
         }
 
-        if (token == null){
-            CMS.debug(method + " auth token is null; agent manual approval required;");
+        /*
+         * this is where we decide whether to let agent do manual approval
+         *  or not
+	 * If auth.instance_id is not set, then request automatically goes
+	 * into queue for agent approval.
```

Comment:
The indentation is not consistent.

Reply:
yes, upgrading to f33 seems to change the tabs in vi. Fixed. thanks.